packages/api/src/domain/medical/__tests__/facility.test.ts

```diff
@@ -79,17 +69,7 @@ describe("isOboEnabled", () => {
         cwOboActive: false,
         cwOboOid: faker.string.uuid(),
       });
-      const resp = isOboEnabled(facility, MedicalDataSource.COMMONWELL);
-      expect(resp).toBeFalsy();
-    });
-
-    it("returns false when obo is active, has oid, but its not initiator only", async () => {
-      const facility = makeFacility({
-        type: FacilityType.initiatorAndResponder,
-        cwOboActive: true,
-        cwOboOid: faker.string.uuid(),
-      });
-      const resp = isOboEnabled(facility, MedicalDataSource.COMMONWELL);
```

Comment:
Removed the last test for both HIEs because we no longer check `if (!isOboFacility(type)) return false;` in the funciton